runtime/moonbase/src/lib.rs

```diff
@@ -634,8 +634,38 @@ impl pallet_author_slot_filter::Config for Runtime {
 	type PotentialAuthors = ParachainStaking;
 }
 
+/// Migration that destroys all of pallet crowdloan rewards' stored data.
+///
+/// This is included in moonbase only so that we can re-test rewards intialization with the new
+/// version of the pallet. This migration makes no attempt to recouperate already-claimed tokens
+/// and it also mints tokens for the new pot.
+///
+/// NOT FOR USE IN VALUE-BEARING NETWORKS!
+pub struct NukeRewardsStorage;
+
+impl frame_support::traits::OnRuntimeUpgrade for NukeRewardsStorage {
+	fn on_runtime_upgrade() -> Weight {
+		// Delete all crowdlaon rewards storage
+		// Supply only the pallet name, no storage item name or map key
+		// This will cause the entire pallet's storage to be deleted
+		frame_support::storage::migration::remove_storage_prefix(b"CrowdloanRewards", &[], &[]);
+
+		// Reset the pot's funds
+		<Balances as frame_support::traits::Currency<AccountId>>::make_free_balance_be(
+			&CrowdloanRewards::account_id(),
+			//TODO is this the right amount to initialize to?
+			3_000_000 * currency::UNIT,
```

Comment:
3000000 seems appropriate, as we want to re-mimic what will happen in moonsilver and moonriver